_docs/sql-reference/sql-commands/021-create-schema.md

```diff
@@ -604,8 +608,32 @@ STATEMENT format displays the schema in a form compatible with the CREATE OR REP
 	 |
 	+--------------------------------------------------------------------------+
 
+### Altering Schema for a Table
+Table schema may be updated using the `ALTER SCHEMA` commands.
+
+The syntax for the command to add (or replace) columns / properties is the following:
+
+    ALTER SCHEMA
+    (FOR TABLE dfs.tmp.nation | PATH '/tmp/schema.json')
+    ADD [OR REPLACE]
+    [COLUMNS (col1 int, col2 varchar)]
+    [PROPERTIES ('prop1'='val1', 'prop2'='val2')]
+
+Add command will fail if column or property with the same name exists, unless `OR REPLACE` keywords are indicated.
+Add command will fail, if the schema file does not exist.
+
+The syntax for the command to remove columns / properties is the following:
```

Comment:
You can remove columns or property with the `ALTER SCHEMA ... REMOVE` command:

Reply:
Thanks, added.